framework/include/st_things/st_things.h

```diff
@@ -49,43 +53,51 @@ extern "C" {
  *            If easy-setup is not done, app can either wait for the user interaction before starting the things stack or
  *            start the things stack directly without waiting for any events(This case is for those things which doesn't
  *            support input capability and for all other unknown cases).
+ *            To use a new json file after initialization, stack should be deinitialized
+ *            and stopped(if its started already).
  * @param[in] json_path Path to Json file which defines a thing. Definition includes the device information,
  *                                       resources and their properties, configuration info for connectivity and easy-setup, etc.
  * @param[out] easysetup_complete Indicates whether easysetup is completed already or not.
  * @return @c 0 on success, otherwise a negative error value
  * @retval #ST_THINGS_ERROR_NONE Successful
  * @retval #ST_THINGS_ERROR_INVALID_PARAMETER Invalid parameter
  * @retval #ST_THINGS_ERROR_OPERATION_FAILED Operation failed
- * @since Tizen RT v1.1
```

Comment:
You should keep this. Don't remove this from all of functions in this file..

Reply:
It is from SDKAPI defined by IoT lab. I'll ask.

Reply:
modified